code/modules/surgery/organs/augments_internal.dm

```diff
@@ -50,19 +50,16 @@
 /obj/item/organ/cyberimp/brain/anti_drop/ui_action_click()
 	active = !active
 	if(active)
-		for(var/obj/item/held_item in owner.held_items)
-			stored_items += held_item
-
 		var/list/hold_list = owner.get_empty_held_indexes()
 		if(LAZYLEN(hold_list) == owner.held_items.len)
 			to_chat(owner, span_notice("You are not holding any items, your hands relax..."))
 			active = FALSE
-			stored_items = list()
-		else
-			for(var/obj/item/stored_item in stored_items)
-				to_chat(owner, span_notice("Your [owner.get_held_index_name(owner.get_held_index_of_item(stored_item))]'s grip tightens."))
-				ADD_TRAIT(stored_item, TRAIT_NODROP, IMPLANT_TRAIT)
-
+			return
+		for(var/obj/item/held_item in owner.held_items)
+			stored_items += held_item
+			to_chat(owner, span_notice("Your [owner.get_held_index_name(owner.get_held_index_of_item(held_item))]'s grip tightens."))
+			ADD_TRAIT(held_item, TRAIT_NODROP, IMPLANT_TRAIT)
+			RegisterSignal(held_item, COMSIG_ITEM_DROPPED, .proc/on_held_item_dropped)
```

Comment:
held_item can be null here.

Reply:
Ok, wait. If you already knew `held_item` could be null if I implemented the above suggestion, why did you suggest it anyway?